src/sources/redis/list.rs

```diff
@@ -46,6 +61,11 @@ impl InputHandler {
     }
 }
 
+async fn backoff_exponential(exp: &u32) {
```

Comment:
This doesn't need to be a reference.

```suggestion
async fn backoff_exponential(exp: u32) {
```